appveyor.yml

```diff
@@ -0,0 +1,31 @@
+version: 1.0.{build}
+image: Visual Studio 2015
+environment:
+  matrix:
+    - CLI_VERSION: 1.0.0-preview2-003131
+    - CLI_VERSION: Latest
+
+install:
```

Comment:
You canemove everything there about install, appveyor has the .net already installed.
The build matrix with `Latest` is optional and not needed in this project.

ref https://docs.microsoft.com/en-us/dotnet/articles/core/tools/using-ci-with-cli